src/lib.rs

```diff
@@ -1,6 +1,27 @@
+use elements::{BlockHash, Txid};
+
+pub mod avg_vbytes;
+pub mod client;
```

Comment:
I don't think we should expose this as a public module, re-export would be better.